side_length: .0587375 #Unit is in meters, equivalent to 2.3125 inches
focal_length: 639
tag_id: 1
